includes/class-freemius.php

```diff
@@ -9985,6 +9985,115 @@ private static function set_account_option_by_module(
             self::$_accounts->set_option( $option_name, $option_value, $store, $network_level_or_blog_id );
         }
 
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since 2.3.1
+         *
+         * @param mixed  $entity
+         * @param string $class
+         *
+         * @return FS_Plugin|FS_User|FS_Site|FS_Plugin_License|FS_Plugin_Plan|FS_Plugin_Tag|FS_Subscription
+         */
+        private static function get_entity( $entity, $class ) {
+            if ( ! is_object( $entity ) || $entity instanceof $class ) {
+                return $entity;
+            }
+
+            return new $class( $entity );
+        }
+
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since 2.3.1
+         *
+         * @param mixed  $entities
+         * @param string $class_name
+         *
+         * @return FS_Plugin[]|FS_User[]|FS_Site[]|FS_Plugin_License[]|FS_Plugin_Plan[]|FS_Plugin_Tag[]|FS_Subscription[]
+         */
+        static function get_entities( $entities, $class_name ) {
+            if ( ! is_array( $entities ) || empty( $entities ) ) {
+                return $entities;
+            }
+
+            // Get first element.
+            $first_array_element = reset( $entities );
+
+            if ( $first_array_element instanceof $class_name ) {
+                /**
+                 * If the first element of the array is an instance of the context class, assume that all other
+                 * elements are instances of the class.
+                 */
+                return $entities;
+            } else if (

```

Comment:
@fajardoleo there is no reason for the `else if` since the prev `if` ends with a `return`